Omelchenko_IherbTest/iherb_cart_modal.py

```diff
@@ -0,0 +1,13 @@
+from selene import browser, by, be
+
+from Omelchenko_iHerbTest.iherb_cart_page import IherbCartPage
+
+
+class IherbCartModal:
+
+    def verify_product_added_to_cart(self):
```

Comment:
what if more than one product?

Reply:
it's by design, in our cart modal can be displayed only product that added, so it will show one product each time you add a product